tests/features/menus_and_blocks/menus.feature

```diff
@@ -1,45 +1,29 @@
-@openy @api
+@openy @api @menutest
 Feature: Menus
   Check that menus items are available and displayed in right regions
 
-  Scenario: Create main menu item and check
-    Given I am logged in as a user with the "Editor" role
-    And I create an item "Give" in the "main" menu
-    When I am an anonymous user
-    And I go to homepage
-    Then I should see "Give" in the "header"
+  Background: Create menu items for test
+    Given I create menu_link_content:
+      | menu_name          | title           | uri                                               | expanded |
+      | main               | BEHAT PROGRAMS  | base:behat-programs                               | 1        |
+      | main               | BEHAT GIVE      | base:behat-give                                   | 1        |
+      | footer-menu-left   | BEHAT LOCATIONS | base:behat-locations                              | 1        |
+      | footer-menu-right  | BEHAT JOIN      | base:behat-join                                   | 1        |
+      | footer-menu-center | BEHAT CAREERS   | base:behat-careers                                | 1        |
+    And I create menu_link_content:
+      | menu_name          | title           | uri                                               | expanded | parent_uri                          | parent_title    | icon_image    |
+      | main               | BEHAT CHILDCARE | base:behat-programs/behat-childcare               | 1        | base:behat-programs                 | BEHAT PROGRAMS  | childcare.png |
+      | main               | BEHAT DAYCARE   | base:behat-programs/behat-childcare/behat-daycare | 1        | base:behat-programs/behat-childcare | BEHAT CHILDCARE |               |
+    Then I am an anonymous user
 
-  Scenario: Create menu item for second level
-    Given I am logged in as a user with the "Editor" role
-    And I create an item "Programs" in the "main" menu
-    And I go to "/admin/structure/menu/manage/main/add"
-    And I fill in the following:
-      | Menu link title | Childcare   |
-      | Link            | /childcare  |
-    And I select "-- Programs" from "Parent link"
-    And I attach the file "childcare.png" to "Icon image"
-    And I press "Save"
-    When I am an anonymous user
-    And I go to the homepage
-    Then I should see "Childcare" in the "dropdown_menu"
-    
-  Scenario: Create footer menu left item and check
```

Comment:
Let's keep at least this scenario as it is checking permissions for Editor. If he is able to access menu section. And attach files to menus.  
If we don't make deletion of items based on names. At least we can delete it just as last item in the list at /admin/structure/menu/manage/footer-menu-left